front/app/api/reports/useDeleteReport.ts

```diff
@@ -0,0 +1,24 @@
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+import fetcher from 'utils/cl-react-query/fetcher';
+import reportsKeys from './keys';
+
+const deletereport = (id: string) =>
```

Comment:
Minor. We could use camelCase to be consistent here and on line 11
```suggestion
const deleteReport = (id: string) =>
```